src/Tizen.NUI/src/public/Utility/Palette.cs

```diff
@@ -204,18 +208,18 @@ public static Palette Generate(PixelBuffer pixelBuffer, Rectangle region)
         /// Returns all of the swatches which make up the palette.
         /// </summary>
         /// <returns>The swatch list</returns>
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        public ReadOnlyCollection<Swatch> GetSwatches()
+        /// <since_tizen> 9 </since_tizen>
+        public IReadOnlyCollection<Swatch> GetSwatches()
         {
-            return new ReadOnlyCollection<Swatch>(swatches);
+            return swatches as IReadOnlyCollection<Swatch>;
```

Comment:
the cast (`as`) is probably not necessary

Reply:
Thank you.